spec/features/user_spec.rb

```diff
@@ -1,12 +1,56 @@
 require 'spec_helper'
 
 feature User do
+  let(:user) { create(:user, name: 'My Name', biography: 'This is my biography!') }
 
-  shared_examples 'admin ability' do
+  describe 'users#show' do
+    let!(:conference) { create(:conference, title: 'My conference') }
+    let!(:unconfirmed_event) { create(:event, state: 'unconfirmed', title: 'This is my test event title!', program: conference.program) }
 
-  end
+    before :each do
+      unconfirmed_event.event_users = [create(:event_user, user: user, event_role: 'submitter')]
+    end
+
+    it 'shows user information' do
+      visit user_path(user)
+      expect(page).to have_content 'My Name'
+      expect(page).to have_content 'This is my biography!'
+      expect(page).to have_selector('img')
+      expect(page).to_not have_content 'My Name presents'
+    end
+
+    it 'does not show event information for unconfirmed events' do
+      expect(page).to_not have_selector('h3', text: 'My Name presents 1 Event:')
```

Comment:
Personally I'm not a big fan of these expect statements ...
(We're doing the same in the old obs tests and we're having a hard time to debug ...)